merkletree/tb.go

```diff
@@ -1,17 +1,26 @@
 package merkletree
 
-import "github.com/coniks-sys/coniks-go/crypto/sign"
+import (
+	"bytes"
+
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+)
 
 type TemporaryBinding struct {
-	Index     []byte
-	Value     []byte
-	Signature []byte
+	// IssuedEpoch would not be included in the serialization.
```

Comment:
Hmm, I think we do want to include the `IssuedEpoch` in the serialization. Otherwise, the server isn't bound to the `IssuedEpoch` by its signature, and the field can be tampered with.